requirements.txt

```diff
@@ -0,0 +1,15 @@
+# Requirements which are not versioned.
```

Comment:
Just FYI - Not that we need to, but I think it's easy enough to create a `requirementx.txt` from an existing environment, in order to pin and reproduce the same environment

Reply:
You can, but I'm not sure we want/need to.  We build with the "latest" version of OS packages so why not the latest version of pip ones.